src/components/notification/notification.component.html

```diff
@@ -36,7 +36,7 @@
     </div>
     <div *ngIf="item.html">
         <div class="sn-content" [innerHTML]="item.html"></div>
-        <div class="icon" *ngIf="item.icon" [innerHTML]="safeSvg"></div>
+        <div class="icon" [ngClass]="{'icon-hover': clickIconToClose}" *ngIf="item.icon" [innerHTML]="safeSvg" (click)="onClickIcon($event)"></div>
```

Comment:
I think it would be cleaner to do `[class.icon-hover]="clickIconToClose"` since it's only the one class being added.